### PR TITLE
ELPP-3199 added peerscout config

### PR DESCRIPTION
**Pull request by de-code**

### CODE REVIEW

File: projects/elife.yaml [hunk 1]
```diff
@@ -918,6 +918,32 @@ reviewer-suggestions:
             1241: 80
         ram: 4096
 
+peerscout:
+    description: PeerScout
```
Comment (giorgiosironi) [inline projects/elife.yaml]: not much of a description like for many other projects :smile_cat:

**Comment by giorgiosironi** [comment]
Would probably add a `DEPRECATED` comment to `reviewer-suggestions`

File: projects/elife.yaml (hunk 1 above)
Comment (de-code) [reply projects/elife.yaml]: Want to keep it consistent ;-)

**Comment by giorgiosironi** [comment]
Merging on green, but not creating new instances until `peerscout-formula` is devoid of `reviewer-suggestions` references.

**Pull request merged**